node/libs/storage/src/block_store/mod.rs

```diff
@@ -75,56 +116,46 @@ pub trait PersistentBlockStore: 'static + fmt::Debug + Send + Sync {
         &self,
         ctx: &ctx::Ctx,
         number: validator::BlockNumber,
-    ) -> ctx::Result<validator::FinalBlock>;
+    ) -> ctx::Result<validator::Block>;
 
     /// Queue the block to be persisted in storage.
     /// `queue_next_block()` may return BEFORE the block is actually persisted,
     /// but if the call succeeded the block is expected to be persisted eventually.
     /// Implementations are only required to accept a block directly after the previous queued
     /// block, starting with `persisted().borrow().next()`.
-    async fn queue_next_block(
-        &self,
-        ctx: &ctx::Ctx,
-        block: validator::FinalBlock,
-    ) -> ctx::Result<()>;
+    async fn queue_next_block(&self, ctx: &ctx::Ctx, block: validator::Block) -> ctx::Result<()>;
```

Comment:
```suggestion
    async fn store_next_block(&self, ctx: &ctx::Ctx, block: validator::Block) -> ctx::Result<()>;
```
I think you already renamed this on the metrics.

Reply:
wdym? This is queue_next_block everywhere (as intended) afaict.